keps/sig-release/1731-publishing-packages/README.md

```diff
@@ -207,25 +426,60 @@ promotion is an optional part of this KEP. As an intermediate solution we can
 also leave the package publishing on the Google side and focus on building them
 before graduating the KEP to GA.
 
-All architectures that are supported by the [package building tool][pkg-gen-kep] should be published.
-This KEP suggests to start with publishing a single supported architecture
-(e.g. `linux/amd64`) and extend that iteratively, when we verify that creating
-all packages for all architectures is fast enough to be done as part of the
-release process. If it turns out this step takes too long, we need to think
-about doing the package building & publishing asynchronous to the release
-process (see also: [Risks](#risks-and-mitigations)).
+At the time of writing this KEP, there are no Go libraries for working with OBS that we could use to integrate directly
+with `krel`. Eventually, we could evaluate if it makes sense to build such a library for our purposes. Until then,
+we'll use `osc` directly (by exec-ing), which also requires adding `osc` to our build images.
 
-### Risks and Mitigations
+[obs-rabbitmq]: https://rabbit.opensuse.org/
 
-- _Risk_: We don't find a proper way to share secrets like the signing key
-  _Mitigation_: Using a third party tool like 1Password
-- _Risk_: Building all the packages for all the distributions and their version takes too long to be done nightly or via cutting the release  
-  _Mitigation_: We do not deliver nightly packages.
+#### Authentication to OBS and User Management
 
-## Design Details
+The concept of API tokens in OBS is very limited and provides access only to a very few endpoints. In other words,
+it's not possible to use API tokens for publishing to OBS. Instead, we need to create some sort of a service account
+to be used when publishing packages. This is one time operation that can be done by SIG Release Leads.
+
+Users are managed manually via the OBS web interface. SIG Release Leads must have access to add/remove users from
+our OBS project. Release Managers should be given read/write access, so they can maintain and create projects and
+packages.
+
+#### How are packages used?
+
+The End Users can configure their systems’ package managers to use
```

Comment:
Nit: I would love to see exact the exact commands for the most popular methods used to consumer our binaries per arch.

Instead of showing the high-level configuration, show the copy/paste migration steps.

Reply:
This is out-of-scope of this KEP for now because it's way too early to decide on that. It'll be part of the communications plan.